src/scripts/scan.sh

```diff
@@ -1,23 +1,25 @@
 #!/usr/bin/env bash
 
 set -e
-VERSION=5.0.1.3006
+VERSION=7.1.0.4889
 SONAR_TOKEN=${SONAR_TOKEN:?Environment variable SONAR_TOKEN is required}
 SCANNER_DIRECTORY=/tmp/cache/scanner
 export SONAR_USER_HOME=$SCANNER_DIRECTORY/.sonar
 OS="linux"
+ARCH="x64"
```

Comment:
It seems CircleCi supports other architectures. We could maybe create an improvement ticket to add the support of ARM: https://circleci.com/docs/using-arm/

Reply:
Yes I agree, we could also support other OS. I will create a ticket